optional_plugins/varianter_pict/avocado_varianter_pict/__init__.py

```diff
@@ -0,0 +1,176 @@
+# Copyright: Red Hat Inc. 2017
+# Authors: Cleber Rosa <crosa@redhat.com>
+
+import hashlib
+import itertools
+import os
+import sys
+
+from avocado.core import exit_codes
+from avocado.core.output import LOG_UI
+from avocado.core.plugin_interfaces import CLI
+from avocado.core.plugin_interfaces import Varianter
+from avocado.core.tree import TreeNode
+from avocado.core.varianter import AvocadoParams
+from avocado.utils import path as utils_path
+from avocado.utils import process
+
+
+class VarianterPictCLI(CLI):
+
+    """
+    Pict based Varianter options
+    """
+
+    name = 'varianter_pict'
+    description = "Pict based Varianter options for the 'run' subcommand"
+
+    def configure(self, parser):
+        try:
+            pict_binary = utils_path.find_command('pict')
+        except utils_path.CmdNotFoundError:
+            pict_binary = None
+
+        for name in ("run", "variants"):  # intentionally ommiting "multiplex"
+            subparser = parser.subcommands.choices.get(name, None)
+            if subparser is None:
+                continue
+            pict = subparser.add_argument_group('pict based varianter options')
+            pict.add_argument('--pict-binary', metavar='PATH',
+                              default=pict_binary,
+                              help=('Where to find the binary version of the '
+                                    'pict tool. Tip: download it from '
+                                    'https://github.com/Microsoft/pict and '
+                                    'run `make` to build it'))
+            pict.add_argument('--pict-parameter-file', metavar='PATH',
+                              help=("Paths to a pict parameter file"))
+            pict.add_argument('--pict-parameter-path', metavar='PATH',
+                              default='/run',
+                              help=('Default path for parameters generated '
+                                    'on the Pict based variants'))
+            pict.add_argument('--pict-order-of-combinations',
+                              metavar='ORDER', type=int, default=2,
+                              help=("Order of combinations. Defaults to "
+                                    "%(default)s, maximum number is specific "
+                                    "to parameter file content"))
+
+    def run(self, args):
+        pass
+
+
+def run_pict(binary, parameter_file, order):
+    cmd = "%s %s /o:%s" % (binary, parameter_file, order)
+    return process.system_output(cmd, shell=True)
+
+
+def parse_pict_output(output):
+    variants = []
+    lines = output.splitlines()
+    header = lines[0]
+    headers = header.split('\t')
+    for line in lines[1:]:
+        variants.append(dict(zip(headers, line.split('\t'))))
+    return (headers, variants)
+
+
+class VarianterPict(Varianter):
+
+    """
+    Processes the pict file into variants
+    """
+
+    name = 'yaml_to_mux'
+    description = 'Multiplexer plugin to parse yaml files to params'
```

Comment:
please update both `name` and `description`.

Reply:
Yep, leftovers ;)